client/client.py

```diff
@@ -29,9 +31,13 @@ def __init__(self, config: Config):
 
         self.config = config.client
 
-        # TODO(alok): delete self.host and self.port, we can use self.config when needed
-        self.host = self.config.host
-        self.port = self.config.port
+        # If we launch a local evaluation, we want the host to be local_host
+        if self.config.game_mode == "local_evaluation":
+            self.host = "local_host"
```

Comment:
I would make this variable local (i.e. just `host`) since it doesn't used later.